server/live_query/redis_live_query.go

```diff
@@ -69,12 +72,36 @@ const (
 type redisLiveQuery struct {
 	// connection pool
 	pool fleet.RedisPool
+	// in memory cache
+	cache           memCache
+	cacheExpiration time.Duration
+
+	logger kitlog.Logger
+}
+
+type memCache struct {
```

Comment:
Though GOLANG has a garbage collector, I have seen cases where memory leaks occur.
I'd recommend looking at the memory at least at sanity level.

Reply:
good call out, we did monitor fargate task memory and there seems to be a very low impact here even with larger (4K) queries